chat.go

```diff
@@ -83,6 +83,67 @@ type ChatCompletionRequest struct {
 	// (Optional)
 	// A unique identifier representing your end-user, which can help OpenAI to monitor and detect abuse
 	User string `json:"user,omitempty"`
+
+	//Additional fields in fork -- Ben Meeker
+
+	// (Optional - default: false)
+	// Return log probabilities of output tokens.
+	LogProbs bool `json:"logprobs,omitempty"`
+
+	// (Optional - default: null)
+	// An integer between 0 and 20 specifying the most likely tokens to return at each token position.
+	// LogProbs MUST be set to TRUE to use this parameter.
+	Top_LogProbs int `json:"top_logprobs,omitempty"`
+
+	// (Optional - default: text)
+	// Specify the format the ChatGPT returns. Compatible with GPT-4o, GPT-4o mini, GPT-4 Turbo, and all GPT-3.5 Turbo models newer that gpt-3.5-turbo-1106
+	// Options
+	// Type: "json_object" to enable JSON mode.
+	// Type: "text" to enable plain text mode.
+	Response_Format *ResponseFormat `json:"response_format,omitempty"`
```

Comment:
nit: Just use camelcase (here and elsewhere)? ResponseFormat

Reply:
I added the underscore naming convention to match the pattern set by Top_P on line 62 of the chat.go file. I figured it was best to follow what was already in place. That way, it matches the API parameters OpenAI accepts more clearly as well. 

I do see your nit side though!! I'm a big camelcase fan 😄